packages/zoltan2/test/partition/CMakeLists.txt

```diff
@@ -636,3 +636,19 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
 
 ##########################################
 
+IF (${PACKAGE_NAME}_ENABLE_Epetra)
```

Comment:
PACKAGE_NAME?  
I think PACKAGE_NAME would be Zoltan2, and I doubt there is a Zoltan2_ENABLE_Epetra flag.
Should it be PROJECT_NAME instead (Trilinos_ENABLE_Epetra)?